src/routes/food.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+
+
+const express = require('express');
+const FoodModel = require('../models/food.js');
+const Food = require('../models/data-collection-class.js');
+
+
+//Initiate new food product
+const food = new Food();
+
```

Comment:
try `const food = new Food(FoodModel);` because then you are telling the code to us this schema to run the CRUDs.